raft/src/main/java/org/apache/kafka/raft/QuorumStateStore.java

```diff
@@ -18,20 +18,33 @@
 
 import java.io.IOException;
 
-public interface ElectionStore {
+/**
+ *  Maintain the save and retrieval of quorum state information, so far only supports
+ *  read and write of election states.
+ */
+public interface QuorumStateStore {
+
+    int UNKNOWN_LEADER_ID = -1;
+    int NOT_VOTED = -1;
+
+    /**
+     * Initialize the quorum state store with given log end offset and epoch.
+     * @param logEndOffsetAndEpoch the valid offset and epoch from metadata log
+     */
+    void initialize(OffsetAndEpoch logEndOffsetAndEpoch) throws IOException;
```

Comment:
This function is not triggered by any non-test callers?

Reply:
Oh good pt, I guess because it's no-op for mock, so didn't catch it.